src/Language/PureScript/Errors.hs

```diff
@@ -746,7 +744,19 @@ prettyPrintSingleError full level showWiki e = flip evalState defaultUnknownMap
             , line "They may be disallowed completely in a future version of the compiler."
             ]
     renderSimpleErrorMessage OverlappingInstances{} = internalError "OverlappingInstances: empty instance list"
-    renderSimpleErrorMessage (NoInstanceFound nm ts) =
+    renderSimpleErrorMessage (NoInstanceFound (Constraint (Qualified (Just (ModuleName [ProperName "Prim"]))
+                                                                     (ProperName "Partial"))
```

Comment:
Is there a declaration in `Language.PureScript.Constants` for the value `Qualified (Just ModuleName [ProperName "Prim"]) (ProperName "Partial")`? Might it make sense to add one, if not?